### ISSUE TITLE
[Wedding] PoC for wedding video action recognition

### ISSUE THREAD

**Issue by allenwu5** [livingbio/action_recognition#1]
**User Story**
做為 Michael，我希望能有 PoC 來了解目前 "机器训练" 進行 "婚纱素材辨识" 的可行性，以及能否滿足 "北京集雅有限公司金剪刀开发服务项目工作说明书" 當中的需求。

**Reference**
> "北京集雅有限公司金剪刀开发服务项目工作说明书"
> 甲方：华人婚庆服务集团股份有限公司
> 乙方：北京集雅科技有限公司

> 前期系统导入服务
> - 婚纱素材辨识强化训练：依甲方提供的样片素材(图片/视频)进行机器训练，强化婚纱拍摄场景的辨识能力

> 甲方工作内容
> - 婚纱素材辨识强化训练：提供欲作为日后应用素材的样片及其原始合成的图片/影片片段

> 乙方工作内容
> - 婚纱素材辨识强化训练：依据甲方所提供的素材进行场景应用的标签设计以及训练。

**Comment by allenwu5** [comment]
Wedding Video Model: https://paper.dropbox.com/doc/Wedding-Video-Model--AcmnFM2dRPZ42AM694E0h~HbAQ-QIT99vxYkwDqW4H2fZtuP

**Comment by allenwu5** [comment]
Training accuracy 可以達到 80 ~ 90%，但 Validation accuracy 卡在 30% 左右，目前提升不上去。

**Comment by allenwu5** [comment]
目前 Validation accuracy 接近 50%：
```
val_acc = 0.4803921568627451
val_micro_recall = 0.4803921568627451
val_macro_f1 = 0.4199522102747909
val_micro_f1 = 0.4803921568627451
val_loss = 1.7144772583798429
top_3_val_acc = 0.7450980392156863
```

**Comment by allenwu5** [comment]
Poc document: https://paper.dropbox.com/doc/Wedding-Video--AeeRXS0~nPxZn8fwiyHeFM4oAQ-GJyvAbfSk5dg3J2zV06Fw
